src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -77,14 +77,19 @@
     private static final int ACTION_SELECT_SOURCE_PATH = 3;
     private static final int ACTION_REQUEST_PASSCODE = 5;
     private static final int ACTION_CONFIRM_PASSCODE = 6;
+    private static final int ACTION_REQUEST_PATTERN = 7;
+    private static final int ACTION_CONFIRM_PATTERN = 8;
 
     private CheckBoxPreference pCode;
+    private CheckBoxPreference patternCode;
```

Comment:
This variable should better not include Code but something like pPattern. I think that **p** character means preference in this case, have a look at the variable just below `Preference pAboutApp`